### ISSUE TITLE
在UTC+8 0:00的时候脚本经常会挂掉

### ISSUE THREAD

**Issue by suica-me** [Dawnnnnnn/bilibili-live-tools#319]
运行系统： Windows 10 2004
Python 版本：3.8.2
log时区：UTC+9
截取的部分log如下：
[log.txt](https://github.com/Dawnnnnnn/bilibili-live-tools/files/4840933/log.txt)
注：此问题并不是每天都发生，但是最近发生的频率似乎越来越高了

看起来是字符解码上的问题
`Task err: UnicodeDecodeError('utf-8', b'\\u5e03\\u4f26\\u5c3c\\u5c14", ` etc.

**Comment by xia-i** [comment]
不是解码的问题，应该是数据不完整

**Comment by suica-me** [comment]
> 应该不是解码的问题，可能是数据不完整

重新看了一下，确实应该是你说的，我记得之前搜的时候有一个相似的issue，当时好像是说没明白是哪出的问题就没有解决

**Comment by Dawnnnnnn** [comment]
啊这，为什么和时间有关系，迷惑....

我记得上个版本加了重试/重连，还会导致脚本挂掉吗

**Comment by suica-me** [comment]
啊这...那我也母鸡啊，我是最新版了（__init__是20200602的），然后就是确实会挂，logger有debug么，我开个debug看看？这两天倒是没有挂，不过前段时间挂的比较多

**Comment by Dawnnnnnn** [comment]
调试日志就是经常输出的黄色的文字，应该在目录下生成了txt的

脚本是在接收到不完整的数据的时候会卡住不动吗？

**Comment by suica-me** [comment]
对..就像 https://github.com/Dawnnnnnn/bilibili-live-tools/issues/319#issuecomment-651470089 所说，数据不完整的时候就pending了

> 调试日志就是经常输出的黄色的文字，应该在目录下生成了txt的
> 
> 脚本是在接收到不完整的数据的时候会卡住不动吗？

**Comment by suica-me** [comment]
然后就是这个..大概是因为0点数据太多了？自动续费的舰长都是0点上吧

> 啊这，为什么和时间有关系，迷惑....

**Comment by Dawnnnnnn** [comment]
鸽置....

**Comment by suica-me** [comment]
¿¿¿¿¿¿¿咕置

**Comment by yawwwwwn** [comment]
> 日志里只有Error的，Error以下的都没写进文件

只有 Error 信息是因为你关了 [[thoroughly_log]](https://github.com/Dawnnnnnn/bilibili-live-tools/blob/b78c88f74861fe11ac05005dfbd6753c60930368/conf/user.conf#L20) 吧。

现在没有舰长抽奖了，所以鸽置。

**Comment by suica-me** [comment]
啊这...我是个傻子dbq
> > 日志里只有Error的，Error以下的都没写进文件
> 
> 只有 Error 信息是因为你关了 [[thoroughly_log]](https://github.com/Dawnnnnnn/bilibili-live-tools/blob/b78c88f74861fe11ac05005dfbd6753c60930368/conf/user.conf#L20) 吧。
> 
了解~~辛苦了
> 现在没有舰长抽奖了，所以鸽置。
